### ISSUE TITLE
Spyder won't launch from an environment transfered with conda-pack

### ISSUE THREAD

**Issue by abakleriche** [spyder-ide/spyder#17151]
I am using Anaconda on Windows 10.
I have a conda environment with Spyder 5.1.5 installed (and also other python packages). 
In this environment I can launch Spyder from the conda prompt and it works.
I want to transfer my environment to a machine which has no internet connection (for security reasons).
I pack my environment with conda-pack and install it on the target machine (conda-unpack)
I launch a conda prompt and activate this environment (conda activate).
When I type "spyder" to launch it from this environment, I get the error : could not find or load the Qt platform plugin "windows" in ""

### Versions
* Spyder version: 5.1.5 None
* Python version: 3.8.10 64-bit
* Qt version: 5.12.9
* PyQt5 version: 5.12.3
* Operating System: Windows 10

### Dependencies
# Mandatory:
atomicwrites >=1.2.0          :  1.4.0 (OK)
chardet >=2.0.0               :  4.0.0 (OK)
cloudpickle >=0.5.0           :  2.0.0 (OK)
cookiecutter >=1.6.0          :  1.7.0 (OK)
diff_match_patch >=20181111   :  20200713 (OK)
intervaltree >=3.0.2          :  3.0.2 (OK)
IPython >=7.6.0               :  7.30.1 (OK)
jedi >=0.17.2;<0.19.0         :  0.18.1 (OK)
jsonschema >=3.2.0            :  4.0.1 (OK)
keyring >=17.0.0              :  18.0.0 (OK)
nbconvert >=4.0               :  6.3.0 (OK)
numpydoc >=0.6.0              :  1.1.0 (OK)
paramiko >=2.4.0              :  2.7.2 (OK)
parso >=0.7.0;<0.9.0          :  0.8.2 (OK)
pexpect >=4.4.0               :  4.8.0 (OK)
pickleshare >=0.4             :  0.7.5 (OK)
psutil >=5.3                  :  5.9.0 (OK)
pygments >=2.0                :  2.10.0 (OK)
pylint >=2.5.0;<2.10.0        :  2.9.6 (OK)
pyls_spyder >=0.4.0           :  0.4.0 (OK)
pylsp >=1.2.2;<1.3.0          :  1.2.3 (OK)
pylsp_black >=1.0.0           :  None (OK)
qdarkstyle =3.0.2             :  3.0.2 (OK)
qstylizer >=0.1.10            :  0.2.1 (OK)
qtawesome >=1.0.2             :  1.0.3 (OK)
qtconsole >=5.1.0             :  5.1.1 (OK)
qtpy >=1.5.0                  :  1.11.2 (OK)
rtree >=0.9.7                 :  0.9.7 (OK)
setuptools >=49.6.0           :  60.2.0 (OK)
sphinx >=0.6.6                :  4.2.0 (OK)
spyder_kernels >=2.1.1;<2.2.0 :  2.1.3 (OK)
textdistance >=4.2.0          :  4.2.1 (OK)
three_merge >=0.1.1           :  0.1.1 (OK)
watchdog >=0.10.3             :  2.1.6 (OK)
zmq >=17                      :  22.3.0 (OK)

# Optional:
cython >=0.21                 :  None (OK)
matplotlib >=2.0.0            :  3.4.3 (OK)
numpy >=1.7                   :  1.19.5 (OK)
pandas >=1.1.1                :  1.3.5 (OK)
scipy >=0.17.0                :  1.7.3 (OK)
sympy >=0.7.3                 :  None (OK)

**Comment by rhkarls** [comment]
As a disclaimer I have not used `conda-pack` myself, but from your comment it is a bit unclear in what order you run the `conda-pack` commands. From the docs it seems the environment should be activated with `source my_env/bin/activate` prior to unpacking. See https://conda.github.io/conda-pack/#commandline-usage. From your error it seems there are some problems with prefix cleanup for Qt.

Also, have you tried cloning the broken environment on the target machine as per the suggestion here: https://stackoverflow.com/a/65119053/752092 ? Although that is not a proper solution (which I guess it more related to `conda-pack` than spyder), perhaps its a work-around for now. 

Probably something like this, after activating and unpacking:

    conda create --name spyder_env_clone --clone spyder_env
    conda activate spyder_env_clone
    spyder

**Comment by abakleriche** [comment]
Thank you for your answer and sorry for beeing unclear in my descrption. Of course, I activated the environment before runing conda-unpack.
I tried the work-around you suggested and it seems to work : Spyder can be launched in the cloned environment.
Since you thought the problem was related to conda-pack, I looked in the conda-pack issues and I found it (https://github.com/conda/conda-pack/issues/181). The solutions to fix tghe problem are to edit and fix the qt.conf file or to clone the environement, just as you said.
Thank you very much for your help !
